packages/seacas/applications/aprepro/CMakeLists.txt

```diff
@@ -19,7 +19,8 @@ TRIBITS_ADD_EXECUTABLE(
 
 install_executable(aprepro)
 
-if(${CMAKE_PROJECT_NAME}_ENABLE_TESTS)
+# Currently doesn't run on windows due to use of sed...
+if (${CMAKE_PROJECT_NAME}_ENABLE_TESTS AND NOT CMAKE_HOST_SYSTEM_NAME STREQUAL "Windows")
```

Comment:
FYI: An alternative is to use `XHOSTTYPE Windows`.  See [here](https://tribits.org/doc/TribitsDevelopersGuide.html#overall-arguments-tribits-add-advanced-test) and [here](https://tribits.org/doc/TribitsDevelopersGuide.html#formal-arguments-tribits-add-test).

The advantage is that if `<Project>_TRACE_ADD_TEST=ON`, then the user (and developer) will see why these tests are not getting added.  Otherwise, they will have to debug CMake code.